### DISCUSSION TITLE
Are the scenarios/ examples folder missing from the distrubution ?

### DISCUSSION THREAD

**Discussion by thalia2809** [groops-devs/groops#283] [Q&A]
Hello 
First time working with groops
I am having a hard time locating the Examples on the github dir
Same with the template file for the GUI
Any help appreciated

**Comment by Waley04** [comment]
@thalia2809,

**|||=== Example for GNSS precise point positioning (PPP) ===|||**
Check [here](https://groops-devs.github.io/groops/html/cookbook.gnssPpp.html)

**|||=== GNSS satellite orbit determination and station network analysis ===|||**
Check [here](https://groops-devs.github.io/groops/html/cookbook.gnssNetwork.html)

=======================
||||||  In general, refer to the documentation [here](https://groops-devs.github.io/groops/html/index.html)


Cheers,

Waley.
